### PR TITLE
add a bool prop, which when true, will disable tag deselection (making it work like a regular single select dropdown)

### PR DESCRIPTION
**Pull request by ReinisV**
add a bool prop, which when true, will disable tag deselection (making it work like a regular single select dropdown).

If this prop is true, then the code skips removing the element, and closes the dropdown.

### CODE REVIEW

**Comment by shentao** [comment]
Could you please describe what is the use case for this?

**Comment by ReinisV** [comment]
Oooh, yeah, sorry. 
I was trying to get this logic https://jsfiddle.net/jqofkzxc/208/ but, for some reason, failing to get it with allow-empty="false", which is why I created a separate prop just to implement this logic. But now, I retried getting what I needed and managed to do it with allow-empty="false", and I understand that allow-empty="false" is basically the same as what I created. 

TLDR: I needed allow-empty="false", which I reimplemented here, because allow-empty="false" did not work for me at first. Now it works, so this pull request can be discarded.

**Comment by shentao** [comment]
👍
